aten/src/ATen/FunctionalizeFallbackKernel.cpp

```diff
@@ -30,7 +30,10 @@
 namespace {
   void functionalizeFallback(const c10::OperatorHandle& op, c10::DispatchKeySet dispatchKeySet, torch::jit::Stack* stack) {
     const auto& schema = op.schema();
-    TORCH_INTERNAL_ASSERT(!schema.hasAnyAliasInfo(), "mutating and aliasing ops should all have codegen'd kernels");
+    TORCH_INTERNAL_ASSERT(
+      !schema.hasAnyAliasInfo(),
+      "mutating and aliasing ops should all have codegen'd kernels. op name: ",
+      op.operator_name().name, ".", op.operator_name().overload_name);
```

Comment:
I'm gonna make this message more actionable, since it's not really an internal assert either (I have an example gist of how to "teach" functionalization about custom mutable ops here that I'll link: https://gist.github.com/bdhirsh/7dadbf6296f8f7d1abcf4c482f438aaa )